content/en/docs/contribute/new-content/open-a-pr.md

```diff
@@ -0,0 +1,498 @@
+---
+title: Opening a pull request
+slug: new-content
+content_template: templates/concept
+weight: 10
+card:
+  name: contribute
+  weight: 40
+---
+
+{{% capture overview %}}
+
+{{< note >}}
+**Code developers**: If you are documenting a new feature for an
+upcoming Kubernetes release, see
+[Document a new feature](/docs/contribute/new-content/new-features/).
+{{< /note >}}
+
+To contribute new content pages or improve existing content pages, open a pull request (PR). Make sure you follow all the requirements in the [Before you begin](#before-you-begin) section.
+
+If your change is small, or you're unfamiliar with git, read [Changes using GitHub](#changes-using-github) to learn how to edit a page.
+
+If your changes are large, read [Work from a local fork](#fork-the-repo) to learn how to make changes locally on your computer.
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+## Changes using GitHub
+
+If you're less experienced with git workflows, here's an easier method of
+opening a pull request.
+
+1.  On the page where you see the issue, select the pencil icon at the top right.
+    You can also scroll to the bottom of the page and select **Edit this page**.
+
```

Comment:
nit: Do you need the extra line between list items?

Reply:
From a markdown perspective, no, but from a markdown _readability_ perspective, I'd argue yes.